desktop/components/credit_card/client/error_handling_form.coffee

```diff
@@ -58,6 +59,7 @@ module.exports = class ErrorHandlingForm extends Backbone.View
       # we either want to override or fallback to the message sent from gravity
       if errorJson.error?
         message = @errors[errorJson.error] || errorJson.error
+        message = message() if ( _.isFunction message)
```

Comment:
Ahh I see, I believe this line is redundant if you just use the one function `errors: ->` and using a function for the message.